cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AsterixQueryGenerator.scala

```diff
@@ -183,6 +183,12 @@ abstract class AsterixQueryGenerator extends IQLGenerator {
       case Relation.inRange => {
         s"$fieldExpr >= ${typeImpl.datetime}('${filter.values(0)}') and $fieldExpr < ${typeImpl.datetime}('${filter.values(1)}')"
       }
+      case Relation.isNull => {
+        s"$fieldExpr is unknown"
+      }
+      case Relation.isNotNull => {
+        s"$fieldExpr is not unknown"
+      }
```

Comment:
Is it better to move the repeated code from line 186 to 191 in different functions to a unified place? E.g., could we handle the `isNull` or `isNotNull` in the entry caller `parseFilterRelation`?

Reply:
@JavierJia Sorry, what do you mean by `JSONRequest` changes? Do you mean `JSONParser`?
For the unification part, I agree, I just didn't know how to do it without changing too much code, I will try to think of one way. Thank you.